pkg/hubble/cell/config.go

```diff
@@ -0,0 +1,287 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Hubble
+
+package hubblecell
+
+import (
+	"fmt"
+	"runtime"
+	"strings"
+
+	"github.com/spf13/pflag"
+	"google.golang.org/protobuf/types/known/fieldmaskpb"
+
+	flowpb "github.com/cilium/cilium/api/v1/flow"
+	ciliumDefaults "github.com/cilium/cilium/pkg/defaults"
+	hubbleDefaults "github.com/cilium/cilium/pkg/hubble/defaults"
+	"github.com/cilium/cilium/pkg/hubble/exporter/exporteroption"
+	"github.com/cilium/cilium/pkg/hubble/observer/observeroption"
+	monitorAPI "github.com/cilium/cilium/pkg/monitor/api"
+	"github.com/cilium/cilium/pkg/time"
+)
+
+type Config struct {
```

Comment:
nit: doesn't need to be exported

Reply:
fixed [here](https://github.com/cilium/cilium/compare/ee4487ba10b7084f884dc9a7715ae79f1e5ab2f9..34c2eeddbfc126528254650ff03453da20ff62a7).